dispatcher/MongoConnect.py

```diff
@@ -304,10 +304,11 @@ def is_timeout(self, doc, t):
             self.log.debug(f'{host} last reported {int(dt)} sec ago')
             ret = ret or True
         if has_ackd is not None and t - has_ackd > self.timeout_take_action:
-            self.log.debug(f'{host} hasn\'t ackd a command from {int(t-has_ackd)} sec ago')
+            self.log.info(f'{host} hasn\'t ackd a command from {int(t-has_ackd)} sec ago')
```

Comment:
Changing from debug to info doesn't have meaningful impact because neither end up in the database, and we're unlikely to start logging at the info level.

Reply:
nevertheless, I feel this is too important to be at debug level. That it does not make a difference for database interactions is irrelevant in this respect

Reply:
Ideally this should be at the `critical` or `fatal` level, because this is a "major" intervention.

Reply:
Great, we are then on the same page. I had assumed that you had set it to `debug` for a reason but I agree, it deserves a higher level logging.